operations/host_spawn.go

```diff
@@ -576,8 +589,12 @@ func hostModifyVolume() cli.Command {
 			defer client.Close()
 
 			opts := restModel.VolumeModifyOptions{
-				NewName: name,
-				Size:    size,
+				NewName:      name,
+				Size:         size,
+				NoExpiration: &noExpiration,
```

Comment:
NoExpiration should only be added if noExpiration == true. I believe, or else if a user tries to modify anything else we will still set NoExpiration to false. It looks like from the host modify operation has a different "expireFlag" for setting noExpiration to false to handle this.

Reply:
This no longer applies now that the interface uses two separate fields. 
(The thinking had been that the route wouldn't clear no expiration when NoExpiration was false. The only way to clear it was to set an expiration. It's better now.)